content/en/sensitive_data_scanner/_index.md

```diff
@@ -28,67 +28,115 @@ Sensitive data, such as credit card numbers, bank routing numbers, and API keys
 
-- [Adding a scanning group](#add-a-scanning-group)
-- [Adding scanning rules](#add-scanning-rules)
-- [Controlling access to events with sensitive data](#control-access-to-events-with-sensitive-data)
+    {{< img src="sensitive_data_scanner/read_write_permissions.png" alt="The compliance permissions sections showing data scanner read and writer permissions" style="width:55%;">}}
+2. Follow the steps to [enable remote configuration][3].
+3. Install the Datadog Agent v7.54 or newer.
 
-You can also [redact sensitive data in tags](#redact-sensitive-data-in-tags).
+[1]: /account_management/rbac/permissions/#compliance
+[2]: /account_management/rbac/
+[3]: /agent/remote_config/?tab=configurationyamlfile#enabling-remote-configuration
+{{% /tab %}}
+{{< /tabs >}}
 
 ### Add a scanning group
 
-A scanning group determines what data to scan. It consists of a query filter and a set of toggles to enable scanning for logs, APM, RUM, and events. See the [Log Search Syntax][4] documentation to learn more about query filters.
+{{< tabs >}}
+{{% tab "In the Cloud" %}}
+A scanning group determines what data to scan. It consists of a query filter and a set of toggles to enable scanning for logs, APM, RUM, and events. See the [Log Search Syntax][2] documentation to learn more about query filters.
 
-For Terraform, see the [Datadog Sensitive Data Scanner group][5] resource.
+For Terraform, see the [Datadog Sensitive Data Scanner group][3] resource.
 
-To set up a scanning group:
+To set up a scanning group, perform the following steps:
 
-1. Navigate to the [Sensitive Data Scanner][6] configuration page. Alternatively, go to **Organization Settings** > **Manage Sensitive Data** and click the **Configuration** tab.
+1. Navigate to the [Sensitive Data Scanner][1] configuration page.
 1. Click **Add scanning group**. Alternatively, click the **Add** dropdown menu on the top right corner of the page and select **Add Scanning Group**.
 1. Enter a query filter for the data you want to scan. At the top, click **APM Spans** to preview the filtered spans. Click **Logs** to see the filtered logs.
 1. Enter a name and description for the group.
 1. Click the toggle buttons to enable Sensitive Data Scanner for the products you want (for example, logs, APM spans, RUM events, and Datadog events).
 1. Click **Create**.
 
+[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration
+[2]: /logs/explorer/search_syntax/
+[3]: https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/sensitive_data_scanner_group
+{{% /tab %}}
+{{% tab "In the Agent" %}}
+<div class="alert alert-warning"><strong>Note</strong>: Sensitive Data Scanner in the Agent supports only one scanning group per organization.</div>
```

Comment:
Alert div implies the bolded Note
```suggestion
<div class="alert alert-warning">Sensitive Data Scanner in the Agent supports only one scanning group per organization.</div>
```

Reply:
I find that people often skip over callout boxes, and I think the bold text helps a little. Or so I like to believe. :)